unitylibs/core/workflow/workflow-acrobat/upload-handler.js

```diff
@@ -46,82 +46,51 @@ export default class UploadHandler {
     return blob;
   }
 
-  async uploadFileToUnityWithRetry(url, blobData, fileType, assetId, signal, chunkNumber = 0) {
-    let retryDelay = 1000;
-    const maxRetries = 4;
-    let error = null;
-    for (let attempt = 1; attempt <= maxRetries; attempt++) {
-      try {
-        const response = await this.uploadFileToUnity(url, blobData, fileType, assetId, signal, chunkNumber);
-        if (response.ok) {
-          this.actionBinder.dispatchAnalyticsEvent('chunk_uploaded', {
-            chunkUploadAttempt: attempt,
-            assetId,
-            chunkNumber,
-            size: `${blobData.size}`,
-            type: `${fileType}`,
-          });
-          return { response, attempt };
-        }
-      } catch (err) {
-        if (err.name === 'AbortError') throw err;
-        error = err;
-      }
-      if (attempt < maxRetries) {
-        const delay = retryDelay;
-        await new Promise((resolve) => { setTimeout(resolve, delay); });
-        retryDelay *= 2;
-      }
+  async afterUploadFileToUnity(assetId, blobData, chunkNumber, fileType, response, attempt) {
```

Comment:
Nit: This is a lot of parameters to pass through the function. Creating an object to pass might help with extensibility and readability.